pkg/network/ovs_test.go

```diff
@@ -0,0 +1,49 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("Testing Ovs", func() {
+	Describe("changeSafeOvs", func() {
+		Context("when it is kept disabled", func() {
+			prev := &opv1alpha1.NetworkAddonsConfigSpec{}
+			new := &opv1alpha1.NetworkAddonsConfigSpec{}
+			It("should pass", func() {
```

Comment:
"should correctly install ovs"

Reply:
this is a unit test we don't install ovs here just validate the function

Reply:
Ok then all good.